tests/test_pg.py

```
@@ -2391,38 +2391,38 @@ def test_self_match_too_few_unique(ctx):
         91710807,
         91714887,
     ]
-    query = sql.select([sql.func.acoustid_compare2(fp, fp, 80)])
-    score = ctx.db.get_fingerprint_db().execute(query).scalar()
+    query = sql.select(sql.func.acoustid_compare2(fp, fp, 80))
+    score = ctx.db.get_fingerprint_db().execute(query).scalar_one()
     assert score >= 1.0
 
 
 @with_script_context
 def test_match_similar_1(ctx):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_match_similar_1`

Add return type annotation: `None`

(ANN201)